dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV DEBIAN_FRONTEND="noninteractive"
 
 RUN apt-get update -qq && apt-get install -y -qq m4 wget time gcc-multilib opam \
         libgtk2.0-dev libgtksourceview2.0-dev \
-        texlive-latex-extra texlive-fonts-recommended hevea \
+        texlive-latex-extra texlive-fonts-recommended \
```

Comment:
Sigh!

Reply:
You can also keep `hevea` in the base deps, I'll do the GC in a separate PR after this one.

Reply:
One extra package is not gonna hurt anyone.

Reply:
Even after #7442 we have this versioning problem but we had all the way along since the first CI incarnations with caching; circle solves this very nicely allowing one to use the checksum of the file, so hopefully gitlab will get something like that some day so we can get rid of manual versioning.